applications/datalinker/values-idfdev.yaml

```diff
@@ -1,2 +1,21 @@
 config:
   slackAlerts: true
+  hipsDefaultDataset: "dp02"
+  hipsDatasets:
+    dp02:
+      url: "https://data-dev.lsst.cloud/api/hips/v2/dp02"
+      paths:
+        - "images/color_gri"
+        - "images/color_riz"
+        - "images/band_u"
+        - "images/band_g"
+        - "images/band_r"
+        - "images/band_i"
+        - "images/band_z"
+        - "images/band_y"
+        - "images/2MASS/Color"
+    dp1:
+      url: "https://data-dev.lsst.cloud/api/hips/v2/dp1"
+      paths:
```

Comment:
I don't know what the right values are here

Reply:
At the moment it's just:

- color_gri
- color_grz
- color_izy
- color_riz
- color_ugr
- color_uug

Presumably at some point they'll get the single band ones up and we'll be ready to cut over to the tree with more bands listed.